src/main/java/edu/ucdavis/dss/ipa/security/Authorizer.java

```diff
@@ -1,21 +1,25 @@
-package edu.ucdavis.dss.ipa.security.authorization;
+package edu.ucdavis.dss.ipa.security;
 
-import edu.ucdavis.dss.ipa.security.Authorization;
 import org.springframework.security.access.AccessDeniedException;
+import org.springframework.stereotype.Service;
+
+import javax.inject.Inject;
 
 /**
  * Created by okadri on 6/22/16.
  */
-public interface Authorizer<T> {
```

Comment:
Same as Authorization, probably good to comment here on what Authorizer does/is responsible for, to help future us.